src/components/overlays/tooltip/Tooltip.vue

```diff
@@ -49,92 +52,117 @@ const {
     <div :class="css.activator">
       <slot name="activator" :open="open" />
     </div>
-    <div
-      v-if="open && !disabled"
-      ref="tooltip"
-      :class="[css.tooltip, tooltipClass]"
-      role="tooltip"
-    >
-      <Transition
-        appear
-        enter-active-class="transition ease-out duration-200"
-        enter-from-class="opacity-0 translate-y-1"
-        enter-to-class="opacity-100 translate-y-0"
-        leave-active-class="transition ease-in duration-150"
-        leave-from-class="opacity-100 translate-y-0"
-        leave-to-class="opacity-0 translate-y-1"
+
+    <Teleport v-if="!disabled" to="body">
+      <div
+        v-if="popperEnter"
+        ref="tooltip"
+        :class="[
+          css.tooltip,
+          tooltipClass,
+          css[`tooltip__${popper?.strategy ?? 'absolute'}`],
+        ]"
+        role="tooltip"
       >
-        <div>
-          <div :class="css.base">
+        <TransitionGroup
+          enter-active-class="transition ease-out duration-200"
+          enter-from-class="opacity-0 translate-y-1"
+          enter-to-class="opacity-100 translate-y-0"
+          leave-active-class="transition ease-in duration-150"
+          leave-from-class="opacity-100 translate-y-0"
+          leave-to-class="opacity-0 translate-y-1"
+          @before-enter="updatePopper()"
+          @after-leave="onPopperLeave()"
+        >
+          <div v-if="open" :class="css.base">
             <slot>
               {{ text }}
             </slot>
           </div>
-          <span v-if="!hideArrow" data-popper-arrow :class="css.arrow" />
-        </div>
-      </Transition>
-    </div>
+          <span
+            v-if="!hideArrow"
+            :class="[css.arrow, { [css.arrow__open]: open }]"
+            data-popper-arrow
+          />
+        </TransitionGroup>
+      </div>
+    </Teleport>
   </div>
 </template>
 
 <style lang="scss" module>
+$arrowSize: 0.625rem;
 .wrapper {
   @apply relative inline-flex;
 
   .activator {
+    width: inherit;
+
     @apply inline;
```

Comment:
So this didn't work for me when I used 0.4.2 packaged. It would show the tooltip, but without any background. Putting `@apply` first actually fixed that.

https://github.com/rotki/ui-library-compat/commit/f78233529fb6be31db42e7e01eb35c8ed5c3f995

Reply:
I am not sure if this is of the way we setup the application or because of vuetify tbh.